src/GroupedMenu/FilteredMenu/index.js

```diff
@@ -32,22 +31,18 @@ class FilteredMenu extends React.Component {
    * @param {object} filter - the filter being toggled
    */
   handleToggleFilter = filter => {
```

Comment:
Why don't you order the filters here using `this.props.filters` as a reference?
That way you don't have to add an extra `order` key to the filter objects?

Reply:
Good idea